modules/ROOT/pages/apikit-odatav4-transform-to-sql-operation.adoc

```diff
@@ -150,16 +134,9 @@ The `transform-to-sql-select` operation supports the following subset of unary a
 
 This operation builds a `WHERE` clause based on the `$search` expression that the system query option provides. The implementation selects all of the elements that have property values that match the condition.
 
-By default, a term matches if the field value contains the term, but it does not match if set as `strict` in the configuration. 
+By default, a term matches if the field value contains the term, but it doesn't match if set as `strict` in the configuration. 
```

Comment:
This is confusing. Does this mean if the `strict search term match` is set to true in the configuration, the value in the field must be an exact match vs fuzzy match? 
Something like:
By default, a term matches if the searched field contains the partial term. If the `strict search term match` is set to `true` in the configuration, the term is considered a match only if the value in the searched field is an exact match.

Reply:
I will leave it up to you. I know I made it more wordy, and maybe it's only me.

Reply:
Yes, you’re right! 
Let's see if you like my sentences :)